plane/src/types/mod.rs

```diff
@@ -219,7 +266,7 @@ pub struct SpawnConfig {
 
     /// The drone pool to use for the connect request.
     #[serde(default)]
-    pub pool: Option<String>,
+    pub pool: Option<DronePoolName>,
```

Comment:
It would probably simplify things to make this just `DronePoolName`, since we already have `#[serde(default)]` and we don't care about the distinction between this being `None` and `Some(DronePoolName::default())`.

Reply:
Oooh, great idea! Vastly simplifies a bunch of stuff and makes the code a lot cleaner :)